acct_mgt_test.py

```diff
@@ -354,6 +361,74 @@ def ms_user_project_remove_role(
     return compare_results(result, success_pattern)
 
 
+def is_moc_quota_empty(moc_quota) -> bool:
+    """This checks to see if an moc quota (name mangled) is empty"""
+    if "Quota" in moc_quota:
+        for item in moc_quota["Quota"]:
+            if moc_quota["Quota"][item] is not None:
+                return False
+    return True
+
+
+def ms_get_moc_quota(acct_mgt_url, project_name, auth_opts=None) -> dict:
+    """gets the moc quota specification (quota name mangled with scope) from the microserver"""
+    cmd = (
+        ["curl", "-X", "GET", "-kv"]
+        + auth_opts
+        + [f"{acct_mgt_url}/projects/{project_name}/quota"]
+    )
+    result = subprocess.run(
+        cmd,
+        stdout=subprocess.PIPE,
+        stderr=subprocess.STDOUT,
+        check=False,
+    )
+
+    moc_quota = {}
+    if result.returncode == 0:
+        line_array = result.stdout.decode("utf-8").split("\n")
+        json_text = ""
+        for line in line_array:
```

Comment:
What does the data returned from `curl` look like htere? It looks like you're interating over all the lines of the output and discarding everything but the last line that starts with a `{`. If the API is returning a JSON document, you could just be able to consume the raw stdout from `curl`.

Reply:
This is off the topic of this PR, of which I intend to keep on the narrow focus of minimal quota support.

Furthermore, it will be dealt with the other PR that is concurrently under review, even though it is really off the narrow focus I had envisioned for that PR as well.